src/NuGet.Core/NuGet.Configuration/Settings/ConfigurationDefaults.cs

```diff
@@ -64,7 +64,7 @@ public IEnumerable<PackageSource> DefaultPackageSources
                         // In a SettingValue representing a package source, the Key represents the name of the package source and the Value its source
                         _defaultPackageSources.Add(new PackageSource(source.GetValueAsPath(),
                             source.Key,
-                            isEnabled: !disabledPackageSources.Any(p => p.Key.Equals(source.Key, StringComparison.CurrentCultureIgnoreCase)),
```

Comment:
It's about adding default sources from `NuGetDefaults.config`  https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior